clang/test/Driver/darwin-embedded-search-paths.c

```diff
@@ -0,0 +1,55 @@
+// UNSUPPORTED: system-windows
+//   Windows is unsupported because we use the Unix path separator `/` in the test.
+
+// Add default directories before running clang to check default
+// search paths.
+// RUN: rm -rf %t && mkdir -p %t
+// RUN: cp -R %S/Inputs/MacOSX15.1.sdk %t/
+// RUN: mkdir -p %t/MacOSX15.1.sdk/usr/include
+// RUN: mkdir -p %t/MacOSX15.1.sdk/usr/include/c++/v1
+// RUN: mkdir -p %t/MacOSX15.1.sdk/usr/local/include
+// RUN: mkdir -p %t/MacOSX15.1.sdk/embedded/usr/include
+// RUN: mkdir -p %t/MacOSX15.1.sdk/embedded/usr/local/include
+
+// Unlike the Darwin driver, the MachO driver doesn't add any framework search paths,
+// only the normal header ones.
```

Comment:
Is there a `CHECK-C-NOT:` you could add to enforce what the comment describes?

Reply:
Yes, good idea let me add that.

Reply:
Actually if we do `-###` then no there isn't a `CHECK_NOT` because the framework search paths aren't on the cc1 line. However, after https://github.com/llvm/llvm-project/pull/120149 then we'll be able to do this so I'll just add the test now.

Reply:
It also means that we can't verify the absolute order of the resulting search path list, but I guess that's ok.